### PR TITLE
Updated changelog to include details from PR #60

### PR DESCRIPTION
**Pull request by cquinn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,13 @@ All notable changes to Corral will be documented in this file. This project adhe
 
 ### Fixed
 
-- Fix  should work if there is no corral.json ([PR #57](https://github.com/ponylang/corral/pull/57))
-- Fix Corral run only looking for binaries on $PATH ([PR #56](https://github.com/ponylang/corral/pull/56))
-- Improved testing and more ([PR #60](https://github.com/ponylang/corral/pull/60))
+- Fix `corral run` should work if there is no corral.json ([PR #57](https://github.com/ponylang/corral/pull/57))
+- Fix `corral run` only looking for binaries on $PATH ([PR #56](https://github.com/ponylang/corral/pull/56))
+- ([PR #60](https://github.com/ponylang/corral/pull/60))
```
Comment (SeanTAllen) [inline CHANGELOG.md]: This should be 1 entry with the same link for each of the sub-items.

Like

- Ensured that fetched bundles are placed in the corral correctly ([PR #60](https://github.com/ponylang/corral/pull/60))

for each one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,13 @@ All notable changes to Corral will be documented in this file. This project adhe
...
+  - #10: Added integration tests
```
Comment (SeanTAllen) [inline CHANGELOG.md]: This isn't user-facing. It shouldn't be in the CHANGELOG,

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,13 @@ All notable changes to Corral will be documented in this file. This project adhe
...
+  - #23: Updated '--directory' to '--bundle_dir', resolved it early, and made sure other dirs are based on it.
```
Comment (SeanTAllen) [inline CHANGELOG.md]: This should be under "CHANGED" section.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,9 +6,13 @@ All notable changes to Corral will be documented in this file. This project adhe
...
+  - #25: Ensured that fetched bundles are placed in the corral correctly.
+  - #29: Verified ProcessMonitor use is correct.
```
Comment (SeanTAllen) [inline CHANGELOG.md]: This doesn't seem like a user-facing change.

File: CHANGELOG.md (hunk 2 above)
Comment (cquinn) [reply CHANGELOG.md]: Ah, ok. I'll remove this one too. I decided the other fixes were not user-facing either.

**Pull request merged**